types/selfsigned.d.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+ declare module "selfsigned" {
```

Comment:
I think it would be better if these were hosted into `@types`. The PR process is actually really simple, I have done it a couple of times.